manifests/4.3/local-storage-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -49,6 +49,7 @@ spec:
     - name: Source Repository
       url: https://github.com/openshift/local-storage-operator
   version: 4.3.0
+  minKubeVersion: 1.14.0
```

Comment:
I'm wondering if we can use 1.16 for 4.3 and 1.17 for 4.4 in order to prevent people from using the wrong manifests.

Reply:
As I understand it - `minKubeVersion` is only designed to have a last minute switch to stop an operator from being installed on a incompatible kube release. I for example when tested this - this indeed prevented operator from being installed but there were no hints in the UI that operator was prevented from being installed because of kube version mis-match.  

In a nutshell - this is not a good mechanism to specify version ranges of kubernetes(or openshift). The manifests are still being displayed to the user because they come from same operatorsource.  I would rather OLM team provided a better mechanism for this. May be index images will solve it.

Reply:
Should be 4.2 requirement noted somewhere in the operator description then?

Reply:
yeah I can update description of 4.2 operator.

Reply:
OLM is working on a better mechanism how to restrict operator version to OCP version, https://coreos.slack.com/archives/C3VS0LV41/p1576081661098700?thread_ts=1576079878.090000&cid=C3VS0LV41

I like it as it is in this PR, let's use `minKubeVersion` to prevent users installing it on 4.1, but then it's up to the user to subscribe to the right channel. Users can install newest 4.3 operator in 4.2, IMO it should work. We must be careful when triaging customer issues, so the operator version is in sync with OCP version and suggest upgrade / downgrade otherwise.